Makefile

```
@@ -67,7 +67,9 @@ test: bin/envtest scheduler-plugins-crd
 
 # Only works with CONTROLLER_VERSION=v2
 .PHONY: test_e2e
-test_e2e: export TEST_MPI_OPERATOR_IMAGE = ${IMAGE_NAME}:${RELEASE_VERSION}
+test_e2e: export TEST_MPI_OPERATOR_IMAGE=${IMAGE_NAME}:${RELEASE_VERSION}
+test_e2e: export TEST_OPENMPI_IMAGE=mpioperator/mpi-pi:${RELEASE_VERSION}-openmpi
+test_e2e: export TEST_INTELMPI_IMAGE=mpioperator/mpi-pi:${RELEASE_VERSION}-intel
 test_e2e: bin/kubectl kind helm images test_images dev_manifest scheduler-plugins-chart
 	SCHEDULER_PLUGINS_VERSION=$(SCHEDULER_PLUGINS_VERSION) go test -v ./test/e2e/...
```

```suggestion
test_e2e: export TEST_INTELMPI_IMAGE=mpioperator/mpi-pi:${RELEASE_VERSION}-intel
test_e2e: export SCHEDULER_PLUGINS_VERSION=${SCHEDULER_PLUGINS_VERSION}
test_e2e: bin/kubectl kind helm images test_images dev_manifest scheduler-plugins-chart
	go test -v ./test/e2e/...
```

Can we use the same way to set up all env vars?

---

done

---

It fails

```
Run make RELEASE_VERSION=test CONTROLLER_VERSION=v2 test_e2e
Makefile:73: *** Recursive variable 'SCHEDULER_PLUGINS_VERSION' references itself (eventually).  Stop.
mkdir -p /home/runner/work/mpi-operator/mpi-operator/bin
```

Perhaps it's not actually necessary to add

---

You're right.